lib/reply.js

```diff
@@ -24,6 +25,7 @@ const {
   kOptions,
   kRouteContext
 } = require('./symbols.js')
+const kWebStreamReadable = Symbol.for('nodejs.stream.readable')
```

Comment:
Should not use non-documented symbol.
We may do a similar check like `typeof payload.getReader === 'function'`.